src/components/Header/index.tsx

```diff
@@ -1,217 +1,262 @@
+    justify-content: space-between;
+    justify-self: center;
+    padding: 1rem;
+    position: fixed;
+    bottom: 0px;
+    left: 0px;
+    width: 100%;
+    z-index: 98;
+    height: 72px;
+    background-color: ${({ theme }) => theme.buttonBlack};
+  `};
+  ${({ theme }) => theme.mediaWidth.upToSmall`
+      height: 60px;
+  `};
+  ${({ theme }) => theme.mediaWidth.upToExtraSmall`
+      padding: 1rem 8px;
+  `};
+`
+
+const HeaderElement = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 8px;
+
+  ${({ theme }) => theme.mediaWidth.upToSmall`
+    align-items: center;
+    width: 100%;
+    justify-content: space-between;
+  `};
+`
+
+const HeaderElementWrap = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 8px;
+  padding: 0px 6px;
+  border-radius: 36px;
+  background-color: ${({ theme }) => theme.background};
+  border: 1px solid ${({ theme }) => theme.background};
+  color: ${({ theme }) => theme.subText};
+  :hover,
+  :focus {
+    background-color: ${({ theme }) => lighten(0.05, theme.background)};
+    border: 1px solid ${({ theme }) => theme.primary};
+  }
+`
+
+const HeaderRow = styled(RowFixed)`
+  ${({ theme }) => theme.mediaWidth.upToMedium`
+   width: 100%;
+  `};
+`
+
+const HeaderLinks = styled(Row)`
+  gap: 4px;
+  justify-content: center;
+
+  ${({ theme }) => theme.mediaWidth.upToLarge`
+    justify-content: flex-end;
+  `};
+
+  ${({ theme }) => theme.mediaWidth.upToExtraSmall`
+     gap: 0px;
+  `}
+`
+
+const IconImage = styled.img<{ isChristmas?: boolean }>`
+  width: 140px;
+  margin-top: ${({ isChristmas }) => (isChristmas ? '-18px' : '1px')};
+
+  ${({ theme, isChristmas }) => theme.mediaWidth.upToSmall`
+    width: 114px;
+    margin-top: ${isChristmas ? '-10px' : '1px'};
+  `};
+
+  ${({ theme }) => theme.mediaWidth.upToExtraSmall`
+    width:100px;
+  `}
+`
+
+const BlogWrapper = styled.span`
+  @media (max-width: 1440px) {
+    display: none;
+  }
+`
+
+const Title = styled(Link)`
+  display: flex;
+  align-items: center;
+  pointer-events: auto;
+  justify-self: flex-start;
+  margin-right: 12px;
+  ${({ theme }) => theme.mediaWidth.upToSmall`
+    justify-self: center;
+  `};
+  :hover {
+    cursor: pointer;
+  }
+`
+
+const LogoIcon = styled.div`
+  transition: transform 0.3s ease;
+
+  :hover {
+    transform: rotate(-5deg);
+  }
+
+  ${({ theme }) => theme.mediaWidth.upToExtraSmall`
+    :hover {
+      transform: rotate(0);
+    }
+  `}
+`
+
+export default function Header() {
+  const { networkInfo, walletKey } = useActiveWeb3React()
+  const isDark = useIsDarkMode()
+  const [holidayMode] = useHolidayMode()
+  const theme = useTheme()
+  const { mixpanelHandler } = useMixpanel()
+  const uptoSmall = useMedia(`(max-width: ${MEDIA_WIDTHS.upToSmall}px)`)
+  const chainSelect = (
+    <MouseoverTooltip
+      text={t`You are currently connected through WalletConnect. If you want to change the connected network, please disconnect your wallet before changing the network.`}
```

Comment:
this one has been removed in #1954. This look like you has resolve merge conflict wrong. Please check again.